lua/entities/gmod_wire_expression2/core/custom/clipboard.lua

```diff
@@ -0,0 +1,14 @@
+util.AddNetworkString( "wire_expression2_set_clipboard_text" )
+
+__e2setcost(100)
+e2function void setClipboardText(string text)
+	-- The maximum allowed length of a single written string is 65532 characters while using net.WriteString
+	-- See https://wiki.facepunch.com/gmod/net.WriteString
+
+	-- This is probably more than sufficient for most people
+	if string.len(text) > 65532 then return self:throw("Exceeded maximum string length for clipboard text", 0) end
```

Comment:
If you're going for really high amounts of characters, I'd recommend using a streamed option or just manually limit this to something a lot lower (for reference, E2 code is streamed in 20kB chunks). I'd suggest at most 8192 characters, and even that's really high.

Also, the throw should return `nil` since the e2function is void.